src/main/java/org/gaul/s3proxy/junit/rules/S3ProxyRule.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2014-2018 Andrew Gaul <andrew@gaul.org>
```

Comment:
Strictly speaking I think this needs to read either "2016-2018 Expedia Inc. and Andrew Gaul" or just "Expedia Inc." until someone else makes a change and gets added to the copyright. We noticed that the current checkstyle rules for the header don't like this though...

Reply:
I assert copyright over the entire project as Apache does for its projects.  Apache does require explicit copyright assignment for substantial changes but not for one-off pull requests.  I do not believe this assignment is necessary in this case as it was not for other contributors who implicitly donated their copyrights.  If you agree with this reasoning please acknowledge that you have the authority to do this.

Reply:
That's fine, the original code is all our original work and we're happy for it to be licensed under an Apache license in the manner you describe above.

Reply:
Sounds good.